README.md

```diff
@@ -53,7 +58,7 @@ In one sentence, explain why.
 
 ## A1.2
 
-«replace this with your answer»
+O(n^3 * G) = O(n^3), where G is number of rules. There are 3 tile types, and while the formula calls for a multiplication by a constant G, the constant is simplified out for big O notation. Thus it is O(n^3).
```

Comment:
I suspect that formula is for time complexity. I believe the PDA will use O(n) for its stack.

# -2